string object_type
float32 detection_confidence
float32 bearing
float32 width
float32 height